src/internal/testutil/identity.go

```diff
@@ -127,6 +130,60 @@ func DoOAuthExchange(t testing.TB, loginURL string) {
 	require.NoError(t, err)
 }
 
+func GetOIDCTokenForTrustedApp(t testing.TB) string {
```

Comment:
This code is all just moved from the OIDC test into testutils, so we can reuse it in the CLI tests.